assets/xml/objects/object_ge1.xml

```diff
@@ -1,63 +1,83 @@
-        <Texture Name="object_ge1_Tex_005038" OutName="tex_005038" Format="ci8" Width="8" Height="16" Offset="0x5038" />
-        <Texture Name="object_ge1_Tex_0050B8" OutName="tex_0050B8" Format="ci8" Width="8" Height="8" Offset="0x50B8" />
-        <Texture Name="object_ge1_Tex_0050F8" OutName="tex_0050F8" Format="rgba16" Width="8" Height="8" Offset="0x50F8" />
-        <Texture Name="object_ge1_TLUT_005178" OutName="tlut_005178" Format="rgba16" Width="16" Height="16" Offset="0x5178" />
-        <Texture Name="object_ge1_Tex_005378" OutName="tex_005378" Format="ci8" Width="16" Height="16" Offset="0x5378" />
-        <DList Name="object_ge1_DL_007F78" Offset="0x7F78" />
-        <DList Name="object_ge1_DL_0085A8" Offset="0x85A8" />
-        <DList Name="object_ge1_DL_0089B0" Offset="0x89B0" />
-        <DList Name="object_ge1_DL_008B58" Offset="0x8B58" />
-        <DList Name="object_ge1_DL_008C88" Offset="0x8C88" />
-        <DList Name="object_ge1_DL_008E80" Offset="0x8E80" />
-        <DList Name="object_ge1_DL_009028" Offset="0x9028" />
-        <DList Name="object_ge1_DL_009158" Offset="0x9158" />
-        <DList Name="object_ge1_DL_009350" Offset="0x9350" />
-        <DList Name="object_ge1_DL_009440" Offset="0x9440" />
-        <DList Name="object_ge1_DL_009570" Offset="0x9570" />
-        <DList Name="object_ge1_DL_0097C0" Offset="0x97C0" />
-        <DList Name="object_ge1_DL_009A20" Offset="0x9A20" />
-        <DList Name="object_ge1_DL_009B50" Offset="0x9B50" />
-        <DList Name="object_ge1_DL_009D18" Offset="0x9D18" />
-        <DList Name="object_ge1_DL_00BB08" Offset="0xBB08" />
-        <DList Name="object_ge1_DL_00BDA0" Offset="0xBDA0" />
-        <DList Name="object_ge1_DL_00C000" Offset="0xC000" />
+        <!-- White-clothed Gerudo Animations -->
+        <Animation Name="gGerudoWhiteStandingHeadBowedAnim" Offset="0x594" /><!-- Original name may be "gb_ie" ("home")? -->
+        <Animation Name="gGerudoWhiteSaluteAnim" Offset="0xABC" /><!-- Original name: "gb_keirei" ("salute/bow") -->
+        <Animation Name="gGerudoWhiteStiffeningAnim" Offset="0xCF8" /><!-- Original name: "gb_kochoku" ("stiffening/rigidity") -->
+        <Animation Name="gGerudoWhiteStiffShiveringAnim" Offset="0x13CC" /><!-- Original name: "gb_kochoku_loop" ("stiffening loop") -->
+        <Animation Name="gGerudoWhiteTrudgingOffAnim" Offset="0x202C" /><!-- Original name: "gb_tobotobo" ("totteringly/trudgingly") -->
+        <Animation Name="gGerudoWhiteGreatBayCutsceneAnim" Offset="0x2148" /><!-- one frame used for each part of Great Bay Temple first entry cutscene. Original name: "gb_twister" -->
+        <Animation Name="gGerudoWhiteExcitedClappingAnim" Offset="0x2954" /><!-- maybe Original name: "gb_wakuwaku", ("excited") -->
+        <Animation Name="gGerudoWhiteUnfoldingArmsAnim" Offset="0x2AD8" /><!-- maybe "ge1_hanasi" ("talk") -->
```

Comment:
Why maybe?

Reply:
@tom-overton wasn't completely sure about this correspondence.

Reply:
These ones I think I believe though, it's the first one that's a bit strange.